library/core/src/main/java/com/google/android/exoplayer2/mediacodec/MediaCodecUtil.java

```diff
@@ -318,7 +318,16 @@ private static boolean isCodecUsableDecoder(android.media.MediaCodecInfo info, S
     }
 
     // Work around https://github.com/google/ExoPlayer/issues/4519.
-    if ("OMX.SEC.mp3.dec".equals(name) && "SM-T530".equals(Util.MODEL)) {
+    if ("OMX.SEC.mp3.dec".equals(name) && ("GT-I9152".equals(Util.MODEL)
```

Comment:
I'm surprised the GT-I9152 isn't in the block below, given this one is a broadcom based device.